pkg/bmc/idrac.go

```diff
@@ -70,3 +70,7 @@ func (a *iDracAccessDetails) DriverInfo(bmcCreds Credentials) map[string]interfa
 func (a *iDracAccessDetails) BootInterface() string {
 	return "ipxe"
 }
+
+func (a *iDracAccessDetails) RAIDInterface() string {
+	return "no-raid"
```

Comment:
Is there a reason not to set it to "agent"?

RAID configuration is going to be added in a driver-independent way, so I'd have thought we'd want to ensure that all drivers can use it somehow.

Reply:
> Is there a reason not to set it to "agent"?

I would like to set it as `no-raid` as default interface

> RAID configuration is going to be added in a driver-independent way, so I'd have thought we'd want to ensure that all drivers can use it somehow.

If vendor driver want to support RAID configuration, they can change `no-raid` to their driver

Reply:
Note that software RAID is vendor-independent.

Reply:
> 
> 
> Note that software RAID is vendor-independent.

Yep. But sofware RAID is out of scope in this version of RAID configuration

Reply:
The default RAID interface for the iDrac driver in Ironic is `idrac-wsman` (unlike the iRMC driver, where the default is `no-raid`). So this change actually disables RAID support in iDrac where before it worked fine, while enabling it in iRMC where before it was disabled.

I think that for all of the non-iRMC drivers you should return an empty string so that it continues to use the driver default instead of overriding it to `no-raid` in every case.

Reply:
Thanks @zaneb 
You're right.  I have just update my PR to set RAID interface of non-iRMC drivers to empty string as you said
Please keep going on :)